tests/test_telemetry.py

```diff
@@ -517,9 +517,12 @@ def test_app_started_product_disabled(self):
         for data in telemetry_data:
             if data["request"]["content"].get("request_type") == "app-started":
                 content = data["request"]["content"]
+                products = content["payload"]["products"]
                 assert (
-                    "products" not in content["payload"]
-                ), "Product information is present telemetry data on app-started event when all products are disabled"
+                    products["appsec"]["enabled"] == False
+                    and products["profiler"]["enabled"] == False
+                    and products["dynamic_instrumentation"]["enabled"] == False
```

Comment:
In .NET we send `app-started` _before_ we know whether `dynamic-instrumentation` is enabled or not, so that typically _isn't_ included in the `app-started` payload (it is sent in a follow "products changed" payload).